lib/improver/cli/estimate_emos_coefficients.py

```diff
@@ -107,18 +152,27 @@ def main(argv=None):
     args = parser.parse_args(args=argv)
 
     # Load Cubes
-    historic_forecast = load_cube(args.historic_filepath)
-    truth = load_cube(args.truth_filepath)
+    historic_forecast = load_cube(args.historic_filepath, allow_none=True)
+    truth = load_cube(args.truth_filepath, allow_none=True)
+
+    combined = (load_cubelist(args.combined_filepath)
+                if args.combined_filepath else None)
+    historic_forecast_dict = (
+        load_json_or_none(args.historic_forecast_identifier))
+    truth_dict = load_json_or_none(args.truth_identifier)
 
     # Process Cube
-    coefficients = process(historic_forecast, truth, args.distribution,
-                           args.cycletime, args.units, args.predictor_of_mean,
-                           args.max_iterations)
+    coefficients = process(historic_forecast, truth, combined,
+                           historic_forecast_dict, truth_dict,
+                           args.distribution, args.cycletime, args.units,
+                           args.predictor_of_mean, args.max_iterations)
     # Save Cube
-    save_netcdf(coefficients, args.output_filepath)
+    if coefficients:
```

Comment:
Might be worth expanding the comment above to say why this is an if statement now

Reply:
This has now been removed.